crates/typst/src/layout/grid/mod.rs

```diff
@@ -210,7 +210,7 @@ pub struct GridElem {
     ///
     /// This can either be a single alignment, an array of alignments
     /// (corresponding to each column) or a function that returns an alignment.
-    /// The function is passed the cells' column and row index, starting at
+    /// The function receives the cells' column and row index, starting from
     /// zero. If set to `{auto}`, the outer alignment is used.
```

Comment:
```suggestion
    /// The function receives the cells' column and row indices, starting from
    /// zero. If set to `{auto}`, the outer alignment is used.
```

Reply:
This was already fixed in the other PR. This PR just needs to be rebased.

Reply:
ah okay. cc @Andrew15-5

Reply:
> This was already fixed in the other PR. This PR just needs to be rebased.

It was? Well, I merged the thing anyway. Didn't even notice it.

Reply:
Oh, yeah, rebase... Forgot it is an option...